### ISSUE TITLE
no posters from IFDB

### ISSUE THREAD

**Issue by benfishbus** [tomfin46/IFDB.bundle#7]
Wasn't getting poster image from IFDB. I'm not very bright with xpath and such, but managed to fix as follows:
```
            # Poster
            poster_url = self.getStringContentFromXPath(html, './/div[' + self.getCssSearchAttr("jrListingMainImage") +
']//img/@src')
```
Changing `//img/@src` to `//a/@href` seemed to work.

**Comment by jt196** [comment]
@tomfin46 can confirm this is a working fix - it's down at the bottom of the init.py file.

**Comment by dabnpits** [comment]
Thanks @benfishbus - The fix worked for me too on Plex 1.25.2.5319-c43dc0277

**Comment by tomfin46** [comment]
Hey all, apologies for not looking into this sooner.

The fix in #8 looks good to me so I'll get that merged in now
